airflow/podman.go

```diff
@@ -175,12 +178,14 @@ func (p *Podman) Stop() error {
 }
 
 func (p *Podman) ExecCommand(containerID, command string) string {
-	command = strings.TrimLeft(command, " ")
-	command = strings.TrimRight(command, " ")
 	execConfig := new(handlers.ExecCreateConfig)
 	execConfig.AttachStdout = true
 	execConfig.AttachStderr = true
-	execConfig.Cmd = strings.Split(command, " ")
+	cmd, err := parseCommand(command)
+	if err != nil {
+		return err.Error()
```

Comment:
Why do we return err.Error() everywhere in this method ?

I mean, if docker daemon is not available or podman not up, we should return an error if this function isn't able to execute a command inside a container, shouldn't we ?

Reply:
I was thinking about doing that 🤔, but I didn't do that just to avoid changing method signatures with re-convergence in mind, but I will add it now, given that we have already updated a few other method signatures.